src/views/RecordingsView.vue

```diff
@@ -382,8 +381,19 @@ export default {
         return a.order - b.order;
       });
       return result;
-    }
-  }
+    },
+    addToListOfTags: function (allTags, tagName, isAutomatic) {
+      if (!allTags[tagName]) {
+        allTags[tagName] = {};
+      }
+
+      if (isAutomatic) {
+        allTags[tagName].automatic = true;
+      } else {
+        allTags[tagName].human = true;
+      }
```

Comment:
You can avoid unnecessary lookups - and make the code a little easier to read - like this:

```js
var tag = allTags[tagName];
if (!tag) {
    allTags[tagName] = tag = {};
}

if (isAutomatic) {
    tag.automatic = true;
} else {
   tag.human = true;
}
```

This works because `tag` is a reference to the value in `allTags`.